Odysee/Controllers/MainViewController.swift

```diff
@@ -191,6 +191,7 @@ class MainViewController: UIViewController, AVPlayerViewControllerDelegate, MFMa
 
     func checkAndShowFirstRun() {
         if !AppDelegate.hasCompletedFirstRun() {
+            Lbryio.deleteAuthToken()
```

Comment:
Don't wanna know what kind of havoc that caused potentially...

Reply:
Yep! 😔 Not sure why keychain items are persistent across installs